### PR TITLE
Added exclude lambda endpoint options

### PR DESCRIPTION
**Pull request by dstasiuk**
It was found that StepFunctionsLocal.jar can read custom lambda service endpoint, **but** if it has been specified the original endpoint will not work.
Even if you specify https://lambda.us-east-1.amazonaws.com for example.

### CODE REVIEW

**Comment by dstasiuk** [comment]
This PR should solve the problem of invoking the lambda function on the remote AWS servers.

File: README.md [hunk 1]
```diff
@@ -193,6 +193,7 @@ You can pass the following environment variables to LocalStack:
 * `DYNAMODB_ERROR_PROBABILITY`: Decimal value between 0.0 (default) and 1.0 to randomly inject `ProvisionedThroughputExceededException` errors into DynamoDB API responses.
 * `DYNAMODB_HEAP_SIZE`: Sets the JAVA EE maximum memory size for dynamodb values are (integer)m for MB, (integer)G for GB default(256m), full table scans require more memory
 * `STEPFUNCTIONS_LAMBDA_ENDPOINT`: URL to use as the lambda service endpoint in step functions. By default this is the localstack lambda endpoint.
```
Comment (whummer) [inline README.md:195]: Thanks for this pull request @dstasiuk ! I'd prefer if we can keep the number of configuration options as low as possible - especially if config options are mutually exclusive, as in this particular example.

Can we please remove the `STEPFUNCTIONS_EXCLUDE_LAMBDA_ENDPOINT` entirely, and introduce a special value (e.g., `"exclude"`) for the existing `STEPFUNCTIONS_LAMBDA_ENDPOINT` variable? I'd propose something like this:

>  * `STEPFUNCTIONS_LAMBDA_ENDPOINT`: URL to use as the lambda service endpoint in step functions. By default this is the localstack lambda endpoint. Set the value to `"exclude"` to skip adding the Lambda service endpoint to the StepFunctions configuration.

Comment (dstasiuk) [reply README.md]: Hi @whummer . Yeap, it makes sense. Fixed!

**Comment by whummer** [comment]
Thanks for updating the PR @dstasiuk 👍

**Comment by gpolyn** [comment]
@dstasiuk (@whummer ) can you link me to any implementation using this new feature? I have a sam local lambda and can ping it from inside my localstack container, but I guess my other config is wrong, because my Step Function cannot find the sam local lambda.

(This workaround might help me withdraw https://github.com/localstack/localstack/issues/3296 .)

**Pull request merged**